src/NServiceBus.AcceptanceTesting/ContextAppender.cs

```diff
@@ -7,17 +7,11 @@
     // This class is written under the assumption that acceptance tests are executed sequentially.
```

Comment:
Correct. Now with the async-aware sticky context every task that is spawned will have (as its descendants) properly assigned `ScenarioContext`